build/android/BUILD.gn

```diff
@@ -103,37 +85,63 @@ action("sign_app") {
   }
 }
 
-copy("brave") {
-  visibility = [ "*" ]
-  deps = []
-  sources = []
-  if (target_cpu == "arm64" || target_cpu == "x64") {
+# Some configurations need mapping file name adjustment to match
+# previous filename on CI.
+needs_mapping_rename = !is_debug
+
+if (needs_mapping_rename) {
+  copy("brave_mappings") {
+    visibility = [ "*" ]
+    deps = []
+    sources = []
+    outputs = []
+
     if (target_android_output_format == "aab") {
-      deps += [ "//chrome/android:monochrome_64_public_bundle" ]
-      sources += [ "$root_out_dir/apks/MonochromePublic64.aab" ]
+      deps += [ "//chrome/android:chrome_public_bundle__dex__r8" ]
+      if (target_cpu == "arm64" || target_cpu == "x64") {
+        sources += [ "$root_out_dir/apks/ChromePublic.aab.mapping" ]
+        outputs += [ "$root_out_dir/apks/MonochromePublic64.aab.mapping" ]
+      } else {
+        sources += [ "$root_out_dir/apks/ChromePublic.aab.mapping" ]
+        outputs += [ "$root_out_dir/apks/MonochromePublic.aab.mapping" ]
+      }
     } else {
-      if (defined(android_app_secondary_abi)) {
-        deps += [ "//chrome/android:monochrome_64_32_public_apk" ]
-        sources += [ "$root_out_dir/apks/MonochromePublic.apk" ]
+      if (target_cpu == "arm" || target_cpu == "x86") {
+        deps += [ "//chrome/android:chrome_public_apk__final_dex__r8" ]
+        sources += [ "$root_out_dir/apks/ChromePublic.apk.mapping" ]
+        outputs += [ "$root_out_dir/apks/MonochromePublic.apk.mapping" ]
       } else {
-        deps += [ "//chrome/android:monochrome_64_public_apk" ]
-        sources += [ "$root_out_dir/apks/MonochromePublic64.apk" ]
+        # apk arm64 or x64
+        deps += [ "//chrome/android:monochrome_64_public_apk__final_dex__r8" ]
+        sources += [ "$root_out_dir/apks/MonochromePublic64.apk.mapping" ]
+        outputs += [ "$root_out_dir/apks/MonochromePublic.apk.mapping" ]
       }
     }
+  }
+}
+
+copy("brave") {
+  visibility = [ "*" ]
```

Comment:
I think you meant `:*` here?

Reply:
yes, this is true
thanks fixed at https://github.com/brave/brave-core/pull/32111/commits/ed7eb1d3cac66579da18e0acc228e8d59a895c71